pkg/component/networking/vpn/seedserver/seedserver.go

```diff
@@ -389,11 +389,8 @@ func (v *vpnSeedServer) podTemplate(configMap *corev1.ConfigMap, secretCAVPN, se
 					},
 					Resources: corev1.ResourceRequirements{
 						Requests: corev1.ResourceList{
-							corev1.ResourceCPU:    resource.MustParse("100m"),
-							corev1.ResourceMemory: resource.MustParse("20Mi"),
-						},
-						Limits: corev1.ResourceList{
-							corev1.ResourceMemory: resource.MustParse("100Mi"),
+							corev1.ResourceCPU:    resource.MustParse("5m"),
+							corev1.ResourceMemory: resource.MustParse("5M"),
```

Comment:
As we talked offline, in the context of https://github.com/gardener/gardener/pull/12030 we should either use `10M` or remove VPA for this container (has to be first if it is recommended at all according to the monitoring data).

Reply:
Container `vpn-seed-server` doesn't have a significant resource consumption. 
If we switch the container to off, we don't have containers to scale for the HA case and can completely set the update mode to off.

Reply:
Thank! I completely agree with the removal of VPA for the vpn-seed-server container. 

Nit: However, I would suggest 10m and 7.5M as static requests:

```
Component vpn-seed-server--vpn-seed-server should consider not using VPA to avoid pointless evictions, at least for side car containers. Recommended static requests are 11m for CPU and 7.5Mi for memory
```